dask/array/ufunc.py

```diff
@@ -114,7 +105,8 @@ def __init__(self, ufunc):
             )
         self._ufunc = ufunc
         self.__name__ = ufunc.__name__
-        copy_docstring(self, ufunc)
+        if isinstance(ufunc, np.ufunc):
```

Comment:
Can you explain why this isnstance is needed now?

Reply:
The name of the `da_frompyfunc` objects are like `frompyfunc-add` and there is no such function in numpy. The docstring from the original function is already being added to these objects in the `frompyfunc` function above.